sashimi/state.py

```diff
@@ -430,7 +465,7 @@ def send_camera_settings(self):
         camera_params = convert_camera_params(self.camera_settings)
         camera_params.trigger_mode = (
             TriggerMode.FREE
-            if self.global_state == GlobalState.PREVIEW
+            if self.global_state == GlobalState.PREVIEW or self.conf["shirashi"] #i added this otherwise during volume aqisition no live image and image saving
```

Comment:
the conf should not be accessed at this point, the State can have boolean variables `camera_enabled`, `laser_enabled`, `scanning_enabled`, which can be used both here (you would check for `scanning_enabled`) and in the GUI construction (we would want to hide the widgets for components that are not enabled)

Reply:
will you work on this with regards to #79 as well or shall i do that?